src/PKSim.Core/Model/PKSimBuildingBlock.cs

```diff
@@ -20,6 +20,13 @@ public interface IPKSimBuildingBlock : ILazyLoadable, IVersionable, IContainer,
       ///    Indicates if a loaded building block was changed and hence needs to be saved
       /// </summary>
       bool HasChanged { get; set; }
+
+      /// <summary>
+      /// A building block might use building block internally to hold structure
+      /// (ExpressionProfile or Population have an internal individual instance).
+      /// In this case, this would be the reference to the parent building block
+      /// </summary>
+      PKSimBuildingBlock OwnedBy { get; set; }
```

Comment:
so here e.g. an Expression profile within an Individual-BB would be OwnedBy this Individual-BB?

Reply:
No. The problem is the following
ExpressionProfile owns an Individual (the organism structure) which olds the enzyme. When changing an expression parameter, it believes that the parameter changed is in the Individual but in fact it's in the ExpressionPRofile whose version should change etc.. So this was required. It is only used when creating the Individual for the expression profile